src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -35,19 +39,37 @@ trainer = pl.Trainer(logger=AimLogger(experiment='experiment_name'))
     {
       title: 'Integrate Hugging Face',
       docsLink: DOCUMENTATIONS.INTEGRATIONS.HUGGING_FACE,
-      code: `from aim.hugging_face import AimCallback
-
-# ...
-aim_callback = AimCallback(repo='/path/to/logs/dir', experiment='mnli')
-trainer = Trainer(
-    model=model,
-    args=training_args,
-    train_dataset=train_dataset if training_args.do_train else None,
-    eval_dataset=eval_dataset if training_args.do_eval else None,
-    callbacks=[aim_callback],
-    # ...
-)
-# ...`,
+      code: `import mlflow
+import transformers
+
+# Set FastTrackML tracking server 
+mlflow.set_tracking_uri("${fasttrack_server}")
+
+def create_huggingface_model(model_path):
+    architecture = "lordtt13/emo-mobilebert"
+    mlflow.transformers.save_model(
```

Comment:
since we don't support save_model/load_model -- is there something like the `callback` approach of the original that we could use? IE, an "mlflow callback"  that could be given to the trainer as in the original, but which uses the provided tracking uri